session.go

```diff
@@ -573,14 +546,18 @@ LOOP:
 }
 
 func (s *session) addTask(pkg interface{}) {
+	f := func() {
+		s.listener.OnMessage(s, pkg)
+		s.incReadPkgNum()
+	}
 	if s.tPool != nil {
 		s.tPool.AddTask(func() {
```

Comment:
replace the parameter of `AddTask` with `f`

Reply:
@wongoo u always do a  great review job.

Reply:
sorry，it was my fault.